deployment.md

```diff
@@ -0,0 +1,41 @@
+# Azure Resources
+
+Create and configure the following resources
+
+## Python
+
+- Storage account
+  - Security + networking -> Access keys, copy key1 value to use in the Function App
+- Cosmos DB
+  - Settings -> Keys blade, copy the Primary Connection String to use in the Function App
+- Function App
+  - Runtime stack: Python v3.9
+  - Version: ~3
+  - Settings -> Configuration blade, create the following application settings:
+    - COSMOS_CONNECTION: Key copied from CosmosDB
+    - CONTEXT_STORAGE_ACCOUNT_CONNECTION: Key copied from Storage Account
+- [YAML file](/python/.vsts-ci.yml)
+  - Create a pipeline with this yml file
+  - Create the following pipeline variables:
+    - `subscription`: The Azure Subscription the resources were created
+      - Ensure the pipeline or Azure DevOps project has access to this subscription
+    - `functionName`: The name of the function app you created
+
+## WebApp
+
+- App Service
+  - Runtime stack: NodeJS >=14
+- [YAML file](.vsts-ci.yml)
+  - Create a pipeline with this yml file
+  - Create the following pipeline variables:
+    - `subscription`: The Azure Subscription the resources were created
+      - Ensure the pipeline or Azure DevOps project has access to this subscription
+    - `webAppName`: The name of the App Service you created
+    - For each exposed function inside the function App (Functions -> Functions blade), get function key in the Developer -> Function Keys blade
+      - `VITE_API_URI`: The url of the functions app (normally https://the_name_of_the_function_app.azurewebsites.net)
+      - `VITE_UPLOAD_FILES_API_KEY`: UploadFile
```

Comment:
Can we set some defaults for these API endpoints? These seem like strange expectations on all deployers.

Reply:
We should be able to use only the master key from the function app, but once I tried and it didn't worked for some reason. I'll try again.
We can't have a default one for the deployed version because each person will get it's own from the azure resource. I guess they were created as multiple because if one get's out it won't affect the other ones.

Reply:
Worked like a charm. I created only one VITE_API_FUNCTIONS_KEY variable and it's the default key from the function app. Now we'll only have: 
- `VITE_API_FUNCTIONS_KEY`: default function host key (available from Functions -> App Keys Tab in Function App instance) and all the API calls will use it.